waspc/src/Wasp/Generator/PackageJsonGenerator.hs

```diff
@@ -53,13 +75,30 @@ genPackageJsonDependencies spec waspDependencies =
               ( dependenciesConflictErrors conflictErrorDeps
                   ++ devDependenciesConflictErrors conflictErrorDeps
               )
+
+getAllPackageJsonDependencies :: AppSpec -> PackageJsonDependencies -> PackageJsonDependencies -> Maybe AllPackageJsonDependencies
+getAllPackageJsonDependencies spec serverWaspDependencies webAppWaspDependencies =
```

Comment:
The caller doesn't need to use the error message, as it will already have been outputted during the generation phase. To support an error message here (that is then unused by the caller) I would need to construct a FullStackPackageJsonDependenciesError structure that combines two underlying PackageJsonDependenciesError structures, all to throw it away in the end.

Reply:
But I will rewrite to use a simple string error message as then it's clearer this indicates a failure.